### DISCUSSION TITLE
Emby Support.

### DISCUSSION THREAD

**Discussion by yocksers** [leinelissen/jellyfin-audio-player#243] [Ideas]
Hello.

This looks to be an awesome app, any chance for it supporting Emby?
Emby and Jellyfin are rather much alike so shouldn't be to difficult to make.

**Comment by leinelissen** [comment]
This is something I've actually been working on and there is support for Emmy in the latest beta. If you'd like to try it, install via TestFlight or download the APK here: https://files.lei.wtf/s/RGYFzgrqzo2fjbe. Let me know how it works for you!

**Reply by yocksers** [reply]
Nice! Been testing it a little and seems to work great.
How ever my testing is limited as i'm still working on setting all my server side stuff up.

Many thanks and keep up the great work!
